.github/workflows/ci-app-e2e-tests.yml

```diff
@@ -0,0 +1,24 @@
+name: CI app E2E tests on main
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+        service_endpoint:
+          description: "The service endpoint URL for the E2E tests"
+          required: true
+          type: string
+        app_name:
+          description: "The application name"
+          required: true
+          type: string
+
+jobs:
+  call-e2e-tests:
+    name: " " # GitHub UI is noisy when calling reusable workflows, so use whitespace for name to reduce noise
```

Comment:
For what its worth, I don't agree with this design decision, eg. the `name: " "` thing. Adding unintuitive whitespace into all our YAML files doesn't seem like the correct way to deal with the problem.

That comment doesn't meaningfully relate to @rylew1's PR though

Reply:
I also dislike the solution. Any suggestions?